cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/ProgramLifecycleService.java

```diff
@@ -241,6 +241,11 @@ public long getProgramRunCount(ProgramId programId) throws Exception {
    * @throws NotFoundException if the given program or program run doesn't exist
    * @throws Exception if authorization failed
    */
+  public RunRecordMeta getRunRecordMeta(ProgramRunId programRunId) throws Exception {
```

Comment:
Why the rename? Is it just a refactoring to have better naming? If it is, just use the refactor rename feature in IntelliJ, it will update all callsite for you. No need to have a pending TODO

Reply:
Correct. Just better naming. Didn't want to mix code refactoring with function changes, thus the todo. But actually there aren't many callsites. So just renamed. 

Original intention was to have
RunRecordMeta getRunRecordMeta()
RunRecord getRunRecord()

but no need to have getRunRecord() as there is no caller.